docs/src/main/sphinx/connector/non-transactional-insert.fragment

```diff
@@ -0,0 +1,13 @@
+Non-transactional INSERT
```

Comment:
IMO this should go to the newly added `jdbc-common-config` fragment. That was the purpose of the fragment. And it's already added to all the places where the config is applicable (since it's implemented in the base-jdbc framework and any connectors based on that inherit the configs too).

Reply:
Maybe as a part of this we can also change the `jdbc-common-config` to use paragraph style documentation of configs as seen on https://trino.io/docs/current/admin/properties-general.html since it allows us to write detailed information instead of one-liners.

Reply:
My issue is that this isn't actually common to all JDBC connectors, only those that support write operations (excluding things like Druid). I think for now it doesn't make sense in the common configurations fragment.

Reply:
Good point. It would make more sense to list it near the config properties though.

Reply:
Agreed - updated

Reply:
``INSERT``